app/config.go

```diff
@@ -37,6 +37,11 @@ type APIConfig struct {
 		Username string `yaml:"username"`
 		Password string `yaml:"password"`
 	} `yaml:"admin"`
+	Captcha struct {
+		Enabled   bool   `yaml:"enabled"`
+		SiteKey   string `yaml:"site-key"`
+		SecretKey string `yaml:"secret-key"`
+	} `yaml:"captcha"`
```

Comment:
Nice to have it on config ! 👍🏻

Reply:
yeah so we can disable it in case. that's also how i made the tests work